delta/app/src/main/resources/app.conf

```diff
@@ -61,7 +61,7 @@ app {
       # the initial contact points
       contact-points = ["127.0.0.1:9042"]
       # the default keyspace
-      keyspace = "delta"
+      keyspace = "delta_1_5"
```

Comment:
why this is modified? If you need to modify it for migrations, you can do it somewhere else.

Reply:
We will need a new keyspace for the migrated data.
And you can't rename keyspaces/tables with Cassandra.
Or do we want to export data and reimport it after data is migrated to keep delta ?

Reply:
but you can override configuration manually on the `MigrationModule`

Reply:
When you restart and run Delta afterwards, you will still need to run it on the new keyspace.
Only migration points to the old keyspace delta as I would like to avoid to run v1.5 by mistake on v1.4 data.

Reply:
I see. Well I guess is fine